sheets-import/server.gs

```diff
@@ -295,25 +278,40 @@ function sendReauthorizationRequest() {
  * by the current user are present; turn it off otherwise.
  */
 function adjustScheduleTrigger() {
+
+  if (TESTING) {
```

Comment:
Since TESTING is true by default, this means triggers cannot be scheduled. I'm not sure this is the best default behavior -- what is your reasoning for including this flag?